src/wayback/tests/test_client.py

```diff
@@ -694,7 +694,7 @@ def test_request_retries(self, requests_mock):
                                               {'text': 'good', 'status_code': 200}])
         session = WaybackSession(retries=2, backoff=0.1)
         response = session.request('GET', 'http://test.com')
-        assert response.ok
+        assert response.is_success
```

Comment:
This change is a bit arbitrary, but I was inspired by httpx's reasoning for the same change: https://www.python-httpx.org/compatibility/#checking-for-success-and-failure-responses

Considering whether we should also do this in the `Memento` class (maybe just a *deprecation* of `.ok` there, though).